test/integration/apiserver/admissionwebhook/admission_test.go

```diff
@@ -138,10 +138,11 @@ var (
 	// admissionExemptResources lists objects which are exempt from admission validation/mutation,
 	// only resources exempted from admission processing by API server should be listed here.
 	admissionExemptResources = map[schema.GroupVersionResource]bool{
-		gvr("admissionregistration.k8s.io", "v1beta1", "mutatingwebhookconfigurations"):       true,
-		gvr("admissionregistration.k8s.io", "v1beta1", "validatingwebhookconfigurations"):     true,
-		gvr("admissionregistration.k8s.io", "v1", "mutatingwebhookconfigurations"):            true,
-		gvr("admissionregistration.k8s.io", "v1", "validatingwebhookconfigurations"):          true,
+		//!TODO: Temporarily disabled for discussion
+		// gvr("admissionregistration.k8s.io", "v1beta1", "mutatingwebhookconfigurations"):       true,
+		// gvr("admissionregistration.k8s.io", "v1beta1", "validatingwebhookconfigurations"):     true,
+		// gvr("admissionregistration.k8s.io", "v1", "mutatingwebhookconfigurations"):            true,
+		// gvr("admissionregistration.k8s.io", "v1", "validatingwebhookconfigurations"):          true,
```

Comment:
ValidatingAdmissionPolicy does not blacklist these types so fails these tests. Should we use a separate list for our ValidatingAdmissionPolicy tests or change VAP to also block web hooks ?